src/it/projects/MJAVADOC-769/src/test/java/mavenbugs/mjavadoc769/RightTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package mavenbugs.mjavadoc769;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+
+import org.junit.jupiter.api.Test;
+
+public class RightTest {
+
+  @Test
+  public void testFoo() throws Exception {
```

Comment:
yes. Otherwise, the scopes of the various dependencies in the project would not make sense, someone would show up saying "you are doing it wrong, you need to change you  scope to bla bla bla" and we would not discuss the actual problem but just why dependency scopes need to be different and then everything works.

This is a *real world* example that highlights why this change makes sense. I did not artificially construct this but literally lifted it 1:1 from Jdbi.